libflux/go/libflux/parser.go

```diff
@@ -4,6 +4,7 @@ package libflux
 // #cgo LDFLAGS: -L. -lflux
 // #include "flux.h"
 // #include <stdlib.h>
+// const int VERSION = 1;
```

Comment:
What is this for?

Reply:
It was for forcing a recompile and I need to remove it.